code/controllers/configuration.dm

```diff
@@ -961,6 +976,12 @@
 			if ("bluespace_revenant_tickrate")
 				bluespace_revenant_tickrate = max(1, text2num(value))
 
+			if ("bluespace_revenant_distortion_max_chance_per_turf")
+				bluespace_revenant_tickrate = max(1, text2num(value))
+
+			if("bluespace_revenant_distortion_maxchance_halfway_point")
+				bluespace_revenant_distortion_maxchance_halfway_point = max(0, text2num(value))
```

Comment:
should probably add all these values to config.txt even if we're keeping the default values

Reply:
Done - I didn't add them initially to keep the config clean, but I can see how it helps. 

There's a deliberate difference between the default code value and our config value for one option (Bloodburner) BTW. The power is disabled from normal play anyhow.